src/Robo/Plugin/Commands/ThemeCommands.php

```diff
@@ -35,8 +35,13 @@ public function __construct()
      */
     public function themeBuild($siteName = 'default'): Result
     {
-        $themeBuildConfiguration = $this->getConfig('theme_build', $siteName);
-        $this->io()->title("theme build");
+        $result = $this->io()->title("theme build");
+        try {
+            $themeBuildConfiguration = $this->getConfig('theme_build', $siteName);
+        } catch (TaskException $e) {
```

Comment:
Will this always be the exception class or do we need to catch anything more broad? e.g. `\Exception`? This is more for my edification than a serious concern.

Reply:
It's our exception so we can make it whatever we want.

https://github.com/ChromaticHQ/chq-robo/blob/ad33f7731f25dccfd916b08e23cf9f151c0af2fd/src/Robo/Plugin/Traits/SitesConfigTrait.php#L85